config/salmon_config.yaml

```diff
@@ -1,38 +1,38 @@
 
-gtf: tests/reference/annotations.gtf
+gtf: /SAN/vyplab/vyplab_reference_genomes/annotation/human/GRCh38/gencode.v40.annotation.gtf 
 
 # Whether to generate custom transcriptome fasta (using gffread)
 # True or False (no quotes)
 generate_fasta: True
 
 # Path to FASTA file of transcript sequences. Ignored if generate_fasta: True
-transcripts_fasta: tests/reference/transcripts.fa
+transcripts_fasta: /SAN/vyplab/vyplab_reference_genomes/sequence/human/gencode/gencode.v40.transcripts.fa
 
 # Path to genome sequence FASTA file. Required if generating index from scratch (for decoy sequences/making transcripts fasta)
-genome_fasta: tests/reference/genome.fa
+genome_fasta: /SAN/vyplab/vyplab_reference_genomes/sequence/human/GRCh38/GRCh38.primary_assembly.genome.fa
 
 # Top level directory storing salmon indices
-salmon_index_dir: "tests/salmon_index/"
+salmon_index_dir: "/SAN/vyplab/vyplab_reference_genomes/salmon/human/GRCh38/full/"
 
 # Name of subdirectory of salmon_index_dir in which to store/find annotation-specific index
-salmon_index_name: gencode_v34
+salmon_index_name: gencode_v40
 
 # kmer-size for salmon index (= minimum acceptable length for valid match)
 # Default: 31 (recommended for >= 75bp reads)
 salmon_kmer_size: 31
 
 # Directory containing input FASTQs
-fastq_dir: tests/pulled_fastq
+fastq_dir: /SAN/vyplab/first_weeks/BDNF_4SU_FULL/analysis/fastp_trimmed/
 # 'pe' for paired-end and 'se' for single-end
 end_type: "pe"
 # Suffix to strip from FASTQs in fastq_dir to get sample names
-fastq1_suffix: "_1.merged.fastq.gz"
+fastq1_suffix: "_1.trimmed.fastq.gz"
 
 # Suffix for 2nd mate of read pairs. Only used if end_type: pe
-fastq2_suffix: "_2.merged.fastq.gz"
+fastq2_suffix: "_2.trimmed.fastq.gz"
 
 # Directory to store Salmon quantifications
-out_dir: tests/salmon/
+out_dir: /SAN/vyplab/first_weeks/BDNF_4SU_FULL/analysis/salmon/
```

Comment:
Would prefer to keep it to the dummy files so that dry runs work locally. Also generally not necessary to push your analysis-specific config to main, can just be copied into the output dir if you want to keep track of it